### DISCUSSION TITLE
Thank y'all so much

### DISCUSSION THREAD

**Discussion by zerobandicoot** [bepvte/bd-addons#27] [General]
I want to thank you all, this plugin was honestly a godsend for me. 
Discord would boot me out of my session and delete all my shortcuts and gifs saved, so this plugin alone made me install BD.
Thank you so much.

I don't know if this is the right place for this, i'll delete it if needed.

**Comment by bepvte** [comment]
Thank you! Btw did the export work?

**Reply by zerobandicoot** [reply]
I'm not sure yet, i have to reinstall discord because it crashes every time i click the gifs button.
Tried repairing BD and moving all plugins outside the plugin folder but nothing happened.

**Reply by bepvte** [reply]
Disable the emote system of bd, and remove emoji related plugins

**Reply by zerobandicoot** [reply]
It worked! Thank you!

And yes, there's not a single gif missing! Though i'm not near the new Discord gif limit, i have just 35-45 gifs rn, not sure if that's what you meant
